### ISSUE TITLE
How to import the output file from MetaPhlan?

### ISSUE THREAD

**Issue by paiboon-aut** [YuLab-SMU/MicrobiotaProcess#58]
Hi, I am a new guy on MicrobiotaProcess package. I learn MicrobiotaProcess package through:  
1. https://www.bioconductor.org/packages/devel/bioc/vignettes/MicrobiotaProcess/inst/doc/Introduction.html
2. https://github.com/YuLab-SMU/MicrobiotaProcess/issues/35
3. https://github.com/YuLab-SMU/MicrobiotaProcess/issues/38.

Form the tutorial, I can follow and run your script in RStudio perfectly!

When I would like to change the data to test the script, I follow this link "https://rdrr.io/github/xiangpin/MicrobiotaProcess/src/R/method-io.R".

However, I cannot import the output file from MetaPhlan to analyze the data. Moreover, I do not know how to set it as mpse1 object like the tutorial.

Would you mind to show me the script how to import the "merged_abundance_table.txt" (attached file) and set it as mpse1 object like the tutorial. 
[merged_abundance_table.txt](https://github.com/YuLab-SMU/MicrobiotaProcess/files/8732388/merged_abundance_table.txt)

Best regards,
paiboon-aut

**Comment by xiangpin** [comment]
First, please check the [`MicrobiotaProcess`](https://www.bioconductor.org/packages/release/bioc/html/MicrobiotaProcess.html) version on your platform.  It is best to use the latest released (`MicrobiotaProcess >=1.8.0`)

```
> library(MicrobiotaProcess)
> mpse <- mp_import_metaphlan(profile='./merged_abundance_table.txt')
> mpse
# A MPSE-tibble (MPSE object) abstraction: 96 × 10
# OTU=16 | Samples=6 | Assays=Abundance | Taxonomy=Kingdom, Phylum, Class, Order, Family, Genus
   OTU      Sample Abundance NCBI_tax_id Kingdom Phylum Class Order Family Genus
   <chr>    <chr>      <dbl> <chr>       <chr>   <chr>  <chr> <chr> <chr>  <chr>
 1 s__Cory… SRS01…       0   2|201174|1… k__Bac… p__Ac… c__A… o__C… f__Co… g__C…
 2 s__Roth… SRS01…       0   2|201174|1… k__Bac… p__Ac… c__A… o__M… f__Mi… g__R…
 3 s__Bact… SRS01…       0   2|976|2006… k__Bac… p__Ba… c__B… o__B… f__Ba… g__B…
 4 s__Prev… SRS01…       0   2|976|2006… k__Bac… p__Ba… c__B… o__B… f__Pr… g__P…
 5 s__Prev… SRS01…       0   2|976|2006… k__Bac… p__Ba… c__B… o__B… f__Pr… g__P…
 6 s__Geme… SRS01…       0   2|1239|910… k__Bac… p__Fi… c__B… o__B… f__un… g__G…
 7 s__Dolo… SRS01…       0   2|1239|910… k__Bac… p__Fi… c__B… o__L… f__Ca… g__D…
 8 s__Lact… SRS01…      80.6 2|1239|910… k__Bac… p__Fi… c__B… o__L… f__La… g__L…
 9 s__Lact… SRS01…      19.4 2|1239|910… k__Bac… p__Fi… c__B… o__L… f__La… g__L…
10 s__Stre… SRS01…       0   2|1239|910… k__Bac… p__Fi… c__B… o__L… f__St… g__S…
# … with 86 more rows
```

And you can see the help info and example of `mp_import_metaphlan` using `?mp_import_metaphlan`

**Comment by paiboon-aut** [comment]
Thank you so much

Best regards,
paiboon-aut
